packages/spindle-tokens/package.json

```
@@ -33,11 +33,12 @@
     "@figma-export/cli": "^4.0.0",
     "@figma-export/types": "^4.0.0",
     "@types/lodash.setwith": "^4.3.6",
+    "@types/jest": "^27.0.0",
```

```suggestion
    "@types/jest": "27.0.0",
```
https://github.com/openameba/spindle/pull/451#issuecomment-1182773071

なるほどです！ここもこちらになりそうでしょうか👀

---

とりあえず今のPRではts関係のpkgのみ外していますがそれ以外のpkgの運用をどうするかも考えないとそうです
結局preserveSemverRanges外すのが一番楽な気もしつつありますが、一旦ts関係のものから`^`外してみてrenovateが想定通り動くか確認できればと思います:pray: